lib/kino_explorer/data_transform_cell.ex

```diff
@@ -21,6 +21,7 @@ defmodule KinoExplorer.DataTransformCell do
     "string",
     "time"
   ]
+  @special_column_types ["list"]
```

Comment:
Why is it a "special" column type? Is it because it is composite?

Reply:
Yep